src/main/resources/templates/ocaml/test/run.sh

```diff
@@ -1,27 +1,96 @@
 #!/bin/bash
 
-OUTPUT_FOLDER="test-reports"
-OUTPUT_FILE="$OUTPUT_FOLDER/results.xml"
-ERROR_LOG="build_errors.txt"
-TIMEOUT="5m"  # Timeout for running all tests
-VERBOSE=false # Set to true to print out compile errors. Be aware that compile errors might show test code.
+# copy code from the assignment or solution to the appropriate test folder
+cp_code()
+{
+    cd "$1" || exit
+    rm ./*.ml >/dev/null 2>&1
+    # shellcheck disable=SC2086
+    ln -s $BUILD_ROOT/$1/src/*.ml ./
+    cd ..
+}
 
+OUTPUT_FILE="test-reports/results.xml"
+BUILD_TIMEOUT="1m"
+TEST_TIMEOUT="10m"
```

Comment:
Should this really be 10 minutes? I know that the individual tests have a timeout by themselves, but I guess the  `$TEST_TIMOUT` should still be something that ensures that the Bamboo servers are not blocked for too long when testing ~700 submissions?

Reply:
If this timeout gets hit, the tests are broken and the student will not get proper feedback.
The exact value doesnt matter as long as it is longer than the sum of the timeouts of the individual tests.
It used to be 5 min but i increased it as qcheck tests are generally slower and i wanted to avoid hitting it.

Reply:
Ok!